src/features/cleaner_last_modified_time.rs

```diff
@@ -11,14 +12,22 @@ pub fn directory_cleaner_based_on_age(
     date: String,
     dry_run: bool,
     report_data: &mut ReportData,
+    paths_to_ignore: &[String],
 ) -> Result<()> {
     let mut del_count: u32 = 0;
     let mut del_size: u64 = 0;
+    // Convert paths_to_ignore to a collection of PathBuf for easier comparison
+    let ignore_set: Vec<PathBuf> = paths_to_ignore.iter().map(PathBuf::from).collect();
     for entry in WalkDir::new(directory).into_iter() {
         match entry {
             Ok(dir) => {
                 let path = dir.path();
 
+                if ignore_set.iter().any(|ignore_path| path.starts_with(ignore_path)) {
```

Comment:
also why are we using startswith?? why can't we compare the actual values since you're parsing to a path